Missing require.userNamespace on some actions tests. Failures on RHEL6

### Version of Singularity:

What version of Singularity are you using? Run:

```
3.5.2
```

### Actual behavior

Test requiring usernamespace is not protected by `require.UserNamespace(t)`. Fails on RHEL6.

```
   --- FAIL: TestSingularityActions/action_URI/trueDockerUserns (2.22s)
                actions_test.go:350: [34mINFO:   [0m Converting OCI blobs to SIF format
                    [34mINFO:   [0m Starting build...
                    Getting image source signatures
                    Copying blob sha256:322973677ef5fb0405c230fed304b2e907f46a1edb921bab269f88547bcf726f
                    Copying config sha256:17b5c69c73d7931d2fb11aa5294104430f93cf142d66f5b9313478ef7fd0926e
                    Writing manifest to image destination
                    Storing signatures
                    2019/12/06 22:55:42  info unpack layer: sha256:322973677ef5fb0405c230fed304b2e907f46a1edb921bab269f88547bcf726f
                    [34mINFO:   [0m Creating SIF file...
                    [34mINFO:   [0m Convert SIF file to sandbox...
                    [91mERROR  : Failed to create user namespace: Invalid argument
                    [0m
                actions_test.go:351: unexpected failure running 'true': <nil>
    FAIL
    coverage: 0.0% of statements
    FAIL    github.com/sylabs/singularity/cmd/singularity   63.217s
```

--- a/cmd/singularity/actions_test.go
+++ b/cmd/singularity/actions_test.go
@@ -20,6 +20,7 @@ import (
 	"testing"
 
 	"github.com/sylabs/singularity/internal/pkg/test"
+	"github.com/sylabs/singularity/internal/pkg/test/tool/require"
 )
 
 // build base image for tests
@@ -345,6 +346,10 @@ func testRunFromURI(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, test.WithoutPrivilege(func(t *testing.T) {
+			if tt.opts.userns {
+				require.UserNamespace(t)
+			}
+
 			_, stderr, exitCode, err := imageExec(t, tt.action, tt.opts, tt.image, tt.argv)
 			if tt.expectSuccess && (exitCode != 0) {
 				t.Log(stderr)

